runtime/bindings/python/src/pyopenvino/graph/passes/manager.cpp

```diff
@@ -39,5 +76,55 @@ void regclass_graph_passes_Manager(py::module m) {
 
     manager.def("set_per_pass_validation", &ManagerWrapper::set_per_pass_validation);
     manager.def("run_passes", &ManagerWrapper::run_passes);
-    manager.def("register_pass", (void (ManagerWrapper::*)(std::string)) & ManagerWrapper::register_pass);
+
+    // TODO: better docs
+    // manager.def(
+    //     "register_pass",
+    //     (void (ManagerWrapper::*)(std::string, py::args args, const py::kwargs& kwargs)) &
+    //         ManagerWrapper::register_pass,
+    // py::arg("pass_name"),
+    // R"(
+    //     Set the type of register pass for pass manager.
+    //     Both optional parameters are related to
+    //     'Serialize' pass.
+
+    //     Parameters
+    //     ----------
+    //     pass_name : str
+    //         String to set the type of pass
+    //     Args:
+    //         param1 : str
+    //             path where .xml file will be saved
+    //         param2 : str
+    //             path where .bin file will be saved
+
+    //     Kwargs:
+    //         xml_path : str
+    //             path where .xml file will be saved
+    //         bin_path : str
+    //             path where .bin file will be saved
+    // )");
+
+    manager.def(
+        // TODO: better docs
+        "register_pass",
+        (void (ManagerWrapper::*)(std::string, const std::string&, const std::string)) & ManagerWrapper::register_pass,
+        py::arg("pass_name"),
+        py::arg("xml_path") = "",
+        py::arg("bin_path") = "",
+        R"(
+        Set the type of register pass for pass manager.
+        Both optional parameters are related to
+        'Serialize' pass. 
```

Comment:
`are related`... you can always create another `.def("register_pass")` without these path arguments thus creating one simple `register_pass` with only name argument and second one for Serialize that would be easier to write without worrying about arguments dispatching on C++ side (at least for now 👀).